code/lib/cli/src/helpers.ts

```diff
@@ -212,10 +215,8 @@ export async function copyComponents(renderer: SupportedRenderers, language: Sup
   };
 
   const destinationPath = await targetPath();
+  await fse.copy(join(getBaseDir(), 'rendererAssets/common'), destinationPath, { overwrite: true });
```

Comment:
Swapped the order here so that a framework/renderer could contain a file that is otherwise covered by the common template files, but the specific one would win.